### PR TITLE
feat: allow multiple microcenter locations

### LINKED ISSUE #386: Select multiple Microcenter Locations
Im fortunate enough to have multiple Microcenter locations available to me, would it be possible to select multiple microcenter locations to monitor?

### PR DESCRIPTION
**Pull request by rortiz9**
### Description

Resolves #386 

In some cases (e.g. Georgia) there are more than one MicroCenter locations nearby, so we'd want to check more than one location.

### Testing

Set MICROCENTER_LOCATION variable to "duluth,marietta" and STORES to "microcenter". Verified in the debug logs that links from both locations were loaded into the Store object for MicroCenter. The bot then checked for cards from MicroCenter successfully.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ Here is a list of variables that you can use to customize your newly copied `.en
...
 | `MAX_PRICE_SERIES_3080` | Maximum price allowed for a match, applies 3080 series cards (does not apply to these sites: Nvidia, Asus, EVGA) | Default: leave empty for no limit, otherwise enter a price (enter whole dollar amounts only, avoid use of: dollar symbols, commas, and periods.) e.g.: `1234` - Cards above `1234` will be skipped. |
 | `MAX_PRICE_SERIES_3090` | Maximum price allowed for a match, applies 3090 series cards (does not apply to these sites: Nvidia, Asus, EVGA) | Default: leave empty for no limit, otherwise enter a price (enter whole dollar amounts only, avoid use of: dollar symbols, commas, and periods.) e.g.: `1234` - Cards above `1234` will be skipped. |
-| `MICROCENTER_LOCATION` | Specific MicroCenter location to search | Default : `web` |
+| `MICROCENTER_LOCATION` | Specific MicroCenter location(s) to search | Comma separated, e.g.: `marietta,duluth` |
```
Comment (jef) [inline README.md:97]: Maybe we can keep the note in here about the default being `web`?

```suggestion
| `MICROCENTER_LOCATION` | Specific MicroCenter location(s) to search | Comma separated, e.g.: `marietta,duluth`, default: `web` |
```

File: src/store/model/microcenter.ts [hunk 2]
```diff
@@ -32,11 +33,65 @@ const microCenterLocationToId: Map<string, string> = new Map([
...
-if (microCenterLocationToId.get(microCenterLocation) === undefined) {
-	storeId = '029';
-} else {
-	storeId = microCenterLocationToId.get(microCenterLocation)!;
+for (const microCenterLocation of microCenterLocations) {
+	if (microCenterLocationToId.get(microCenterLocation) === undefined) {
+		const logString = `No MicroCenter location named ${microCenterLocation}, skipping.`;
+		logger.warn(logString);
+	} else {
+		storeId = microCenterLocationToId.get(microCenterLocation)!;
```
Comment (jef) [inline src/store/model/microcenter.ts:43]: What does the `!` mean here?

**Review by jef — COMMENTED** [review]
Overall, looks pretty good. Just a comment and question.

Thanks for doing this!

File: src/store/model/microcenter.ts (hunk 2 above)
Comment (rortiz9) [reply src/store/model/microcenter.ts:43]: Non-null assertion. It was there before I moved this line, not sure if it's still needed.

**Review by jef — APPROVED** [review]
Not sure why type of rate limiting Microcenter has, but hopefully it doesn't have a low threshold :sweat_smile: This is quite a lot of extra links!

Thanks for doing this! :shipit:

**Pull request merged**